src/main/java/org/broadinstitute/hellbender/tools/walkers/readorientation/AltSiteRecord.java

```diff
@@ -26,6 +28,8 @@
     private int depth;
     private Nucleotide altAllele;
 
+    public static final String SAMPLE_METADATA_TAG = "SAMPLE";
```

Comment:
It's weird that this isn't in htsjdk since the key _is_ in the spec, but the VCFHeader code is sort of a mess.

Reply:
I moved all the `SAMPLE_METADATA_TAG`s to `TableUtils`, which is a bit of an improvement.